### ISSUE TITLE
Recorded website missing external media (from different domain)

### ISSUE THREAD

**Issue by claudiobizzotto** [webrecorder/pywb#693]
## Describe the bug

(Not sure if this is a bug or a feature request.) The web recorder doesn't seem to save local copies of media, such as images, whose sources are on a different domain than the original website being archived.

## Steps to reproduce the bug

```shell
wb-manager init my-web-archive
wayback --record --live -a --auto-interval 10
```
I then open a web browser at `http://localhost:8080/my-web-archive/record/<url-to-be-recorded>`.

## Expected behavior

I would expect a copy of each media file (in this case, each image), regardless of origin (same domain or different domain), to be available locally.

## Environment

- OS: Debian 11
- Browser: Firefox (version 91)

**Comment by ikreymer** [comment]
Yes, that is the intended, when you load `http://localhost:8080/my-web-archive/record/<url-to-be-recorded>` in the browser, all the URLs loaded on the page, regardless of their origin, should be recorded.

Can't really say more about what went wrong without looking at the particular URL or the WARCs
